digdag-standards/src/test/java/io/digdag/standards/operator/td/TdOperatorFactoryTest.java

```diff
@@ -101,6 +101,56 @@ public void testTDJobRequestParamsWithEngineVersion()
         assertEquals("stable", jobRequest.getEngineVersion().get().toString());
     }
 
+    @Test
+    public void testTDJobRequestParamsWithHiveEngineVersion()
+            throws Exception
+    {
+        Path projectPath = Paths.get("").normalize().toAbsolutePath();
+
+        Config config = newConfig()
+                .set("database", "testdb")
+                .set("query", "select 1")
+                .set("engine", "hive")
+                .set("hive_engine_version", "stable")
```

Comment:
How about adding `engine_version` to check the new parameter expectedly overrides it?